1-exercises/A-this/exercise-3.js

```diff
@@ -8,8 +8,15 @@
 
 let bankAccount = {
   currentAmount: 100,
-  deposit: function (amount) {},
-  withdraw: function (amount) {},
+  deposit: function (amount) {
+    return this.currentAmount += amount;
```

Comment:
similar to the above; if you're just mutating a property you don't need to `return` it (although, you may want to return the mutated property instead)